src/main/java/org/killbill/billing/plugin/adyen/core/AdyenConfigPropertiesConfigurationHandler.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2014-2016 Groupon, Inc
+ * Copyright 2014-2016 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.adyen.core;
+
+import java.util.Properties;
+
+import org.killbill.billing.osgi.libs.killbill.OSGIKillbillAPI;
+import org.killbill.billing.osgi.libs.killbill.OSGIKillbillLogService;
+import org.killbill.billing.plugin.adyen.client.AdyenConfigProperties;
+import org.killbill.billing.plugin.api.notification.PluginTenantConfigurableConfigurationHandler;
+
+public class AdyenConfigPropertiesConfigurationHandler extends PluginTenantConfigurableConfigurationHandler<AdyenConfigProperties> {
```

Comment:
The name of this class looks a lit bit too long. Not sure if we care that much though.